mbed_devices/mbed_tools/cli.py

```diff
@@ -11,7 +11,8 @@ def _build_output(devices: Iterable[Device]) -> str:
     devices_data = []
     for device in devices:
         mount_points = ", ".join(str(mount_point) for mount_point in device.mount_points)
-        devices_data.append([device.mbed_target.platform_name, device.serial_number, device.serial_port, mount_points])
+        platform_name = device.mbed_target.platform_name if device.mbed_target else "UNKNOWN"
```

Comment:
Could we define "UNKNOWN" as a constant which could be used elsewhere in the project?

Reply:
should we not do the same for serial?

Reply:
Good point with serial, I'll change that. I don't think it's worth having such constant yet. Will definitely keep that in mind when looking at presentation layer, is that ok?

Reply:
Fixed https://github.com/ARMmbed/mbed-devices/pull/24/commits/541a0092c3f24c0ce1974b7b5c7c73652ba303e6.